vendor/blanket-require.js

```diff
@@ -89,18 +90,24 @@ if (blanket.options('enableCoverage')) {
  * entries and annotate them so blanket will indicate their non-coverage
  */
 moduleLoaderFinish = function() {
-  if (blanket.options('enableCoverage')) {
-    for (var moduleName in requirejs.entries) {
-      if (typeof(seen[moduleName]) === 'undefined') {
-        seen[moduleName] = true;
-        if (!shouldExclude(moduleName)) {
-          try {
-            blanketLoader(moduleName);
-          } catch (err) {
-            console.log(err);
-          }
-        }
-      }
-    }
-  }
+  // if (blanket.options('enableCoverage')) {
```

Comment:
@jschilli The purpose of this was to report coverage (as 0%) on files that weren't part of the required files when running the app/tests - e.g. a util class that wasn't imported.  I'm not sure if that's handled, but it's such a low runner that I'm also not concerned if we don't handle it.